src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -409,9 +409,6 @@ internal override bool CanChangeActiveContextDocument
             }
         }
 
-        internal override bool CanRenameFilesDuringCodeActions(CodeAnalysis.Project project)
-            => !IsCPSProject(project);
-
         internal bool IsCPSProject(CodeAnalysis.Project project)
```

Comment:
The comment in IsCPSProject implies it's only being used for this rename case. Can it be deleted too?

Reply:
It can't be deleted, but I can update the comment. It's used by [this code](http://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/DesignerAttribute/DesignerAttributeIncrementalAnalyzer.cs,09f9847477df4117) in DesignerAttribute

Reply:
def update the comment thanks :)